stablehlo/dialect/Version.h

```diff
@@ -38,7 +38,7 @@ class Version {
   static FailureOr<Version> fromString(llvm::StringRef versionRef);
 
   /// Return a Version representing the current VHLO dialect version.
-  static Version getCurrentVersion() { return Version(0, 15, 1); }
+  static Version getCurrentVersion() { return Version(0, 15, 2); }
```

Comment:
Leaving a comment here to prevent an accidental merge.

Reply:
Downstream integrate has landed in https://github.com/tensorflow/mlir-hlo/commit/39e79cdc3f5c4a68d025d94f34ef892b47cbb0bf.